src/MenuItem.react.js

```diff
@@ -33,7 +33,7 @@ const MenuItem = React.createClass({
           'active': active,
           'disabled': disabled,
         }, className)}>
-        <a href="#" onClick={this._handleClick}>
+        <a style={{ cursor: 'pointer' }} onClick={this._handleClick}>
```

Comment:
@ericgio branch was updated.
https://github.com/ericgio/react-bootstrap-typeahead/pull/97/commits/b42ffa2f4aeba50e14f3cbe47c3b8e65fa2a4ce5

thank you sir